packages/shared/src/domain/address/zip.ts

```diff
@@ -1,18 +1,3 @@
-function isValidZipCode(zipCode: string): boolean {
-  if (!zipCode) return false;
-  if (zipCode.length === 0) return false;
-  if (!zipCode.match(/^[0-9-]+$/)) return false;
-  return true;
-}
-
-export function isValidZipCodeStrict(zipCode: string): boolean {
-  const isValid = isValidZipCode(zipCode);
-  if (!isValid) return false;
-  if (zipCode.length !== 5) return false;
-  return true;
-}
-
-// TODO 2330 Move/merge this to normalizeZipCode
```

Comment:
Good catch! Someone forgot to remove these.

Since we're here, can you also update the function names so they don't have `new` any more, please?

Reply:
addressed